ontology/yaml/resources/HVAC/entity_types/HX.yaml

```diff
@@ -146,7 +146,7 @@ HX_CWRWISOVM_TWRWISOVM_CWTWRTM:
   - condensing_return_water_temperature_sensor
   - outside_condensing_loop_return_water_temperature_sensor
 
-HX_CWRWISOVPM_TWRWISOVPM_CWTWRTM:
+HX_CWRWISOVPM_OCWRWISOVPM:
```

Comment:
please retain the CWTWRTM (it is meant to capture the individual fields defined)